app/src/Components/Project/ProjectAttachments.tsx

```diff
@@ -17,10 +21,20 @@ const AttachmentDetails = styled.Text`
   font-size: 16px;
   margin-left: 12px;
 `
-
-const ProjectAttachments: FC<ProjectAttachmentsProps> = ({ icon, details }) => {
+/**
+ * Displays the attachment button
+ * @param {string} url of project scope url
```

Comment:
Added a comment to your message above but they get lost really easily on this! A warning appears in the ESlint when I add icon and details saying that they are unused parameters...maybe because they're being used within the AttachmentButton JSdocs doesn't read them the say way 🤔

Reply:
I think that's because what we're actually passing in here is one parameter, a `props` parameter of type `ProjectAttachmentProps`, that has three properties:
a `React.ReactNode` - `props.icon`,
a `string` - `props.details`
and another `string` - `props.url`

Took a bit of googling to work out how to document this in JSDocs, but there's [an example format here](https://github.com/Scottish-Tech-Army/Volunteer-app/pull/109/files#diff-6e3110bb4f79fc306bb078ffce7e71a02c2ddb333c5539219e0b5fb93a7ddd1eR20) you can copy (lines 20 - 26, although obviously that has more lines than you'll need here).  Essentially you put a `param` line for the `props` object first.  Then also one for each property e.g. `props.icon`   

(It's one of the recommended JSDocs approaches and was the preferred way of doing it when I asked the team on Slack + passes the ES Lint tests.  NB don't be thrown by the square brackets in that example -- that's to show a parameter is optional, which you won't need here.)